src/pexshell.rs

```diff
@@ -1,7 +1,7 @@
 use crate::{
     argparse,
     cli::Console,
-    config::{self, Config, Manager as ConfigManager, Provider},
+    config::{Config, Manager as ConfigManager, Provider},
```

Comment:
If we just call it `Provider` it's not clear what it's providing. Try doing something similar to `ConfigManager`:

```suggestion
    config::{Config, Manager as ConfigManager, Provider as ConfigProvider},
```